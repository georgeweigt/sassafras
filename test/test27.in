data ;
input x $ y ;
datalines ;
a 1
a 2
b 3
b 4
c 5
c 6
;

proc print ;
var x ;
