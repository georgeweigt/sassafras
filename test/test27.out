                                   Obs     X
                                     1     a
                                     2     a
                                     3     b
                                     4     b
                                     5     c
                                     6     c

